### PR TITLE
Fixed typo in mender/cli/cli.go

### PR DESCRIPTION
**Pull request by NilsOlavKJohansen**
changelog: none

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure that all commits have a [`Changelog`](../CONTRIBUTING.md#changelog-tags) tag. If nothing should be added to the Changelog, add a `Changelog: None` tag. If there is a change, add `Changelog: Commit`, or `Changelog: Title`, depending on what should be included in the changelog.

- [x] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by oleorhagen** [comment]
@NilsOlavKJohansen it seems to me that you did not sign your commit properly.

This you can see for yourself, by clicking on the `ci/gitlab` job (Details). and then looking through the build log.

**Comment by lluiscampos** [comment]
@oleorhagen @NilsOlavKJohansen You should stop the `mender-qa` pipeline (expensive) when a PR does not actually change behavior, like this one only changing a comment in the source code.

**Comment by oleorhagen** [comment]
> @oleorhagen @NilsOlavKJohansen You should stop the `mender-qa` pipeline (expensive) when a PR does not actually change behavior, like this one only changing a comment in the source code.

Yes, we're getting to it. But this one is for instructional purposes :)

File: cli/cli.go [hunk 1]
```diff
@@ -534,7 +534,7 @@ func upgradeHelpPrinter(defaultPrinter func(w io.Writer, templ string, data inte
 	w io.Writer, templ string, data interface{}) {
 	// Applies the ordinary help printer with column post processing
 	return func(stdout io.Writer, templ string, data interface{}) {
-		// Need at least 10 characters for lastr column in order to
+		// Need at least 10 characters for last column in order to
```
Comment (merlin-northern) [inline cli/cli.go:537]: to be very mean, should it be "the last" ?

**Pull request merged**